### DISCUSSION TITLE
Trying to install

### DISCUSSION THREAD

**Discussion by bacardiiii** [BJReplay/ha-solcast-solar#7] [General]
Hi

I removed my old solcast repo from my integrations and then added the new repo. I have tried adding it under settings, devices and services but i cant find it. If i check Hacs frontend i can see the solcast lovelace repo. what am i doing wrong? 

I solved my problem. i clicked the install link which took me to Hacs and i added it. However it installed as plugin. I had to uninstall it then manually add it as an integration,

**Comment by icefest** [comment]
Did it keep all your old sensor names, or did you have to do any reconfiguring?

**Comment by bacardiiii** [comment]
It used the same entity names. So my automations just worked

**Comment by BJReplay** [comment]
> I solved my problem. i clicked the install link which took me to Hacs and i added it. However it installed as plugin. I had to uninstall it then manually add it as an integration,

For a short while, I had the same link as oziee, which was for a plugin link.  I was fixed not too long after that as an integration link.

As noted in the updated readme, you need to install manually until it is added back into HACS.
